File: Baekjoon/Dynamic Programming/9461.kt
fun main(args: Array<String>)
{
    val T = readLine()!!.toInt()
    val arr = Array(101){0L}
    arr[1] = 1
    arr[2] = 1

    for (j in 3..100){
        arr[j] = arr[j-3] + arr[j-2]
    }

    for (i in 1..T){
        val n = readLine()!!.toInt()
        println(arr[n])
    }
}

File: Baekjoon/Dynamic Programming/9461.py
T = int(input())
arr = [0]*101
arr[1], arr[2], arr[3] = 1, 1, 1

for _ in range(T):
    n = int(input())

    if n > 3:
        for i in range(4,n+1):
            arr[i] = arr[i-3] + arr[i-2]
    print(arr[n])